repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -75,6 +77,7 @@ import scala.io.Source
   */
 class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode, desiredOutputStream: ServletOutputStream)
   extends javax.servlet.http.HttpServletResponseWrapper(originalResponse) with HeaderInteractor {
+  private val headerWarningLogger = LoggerFactory.getLogger(classOf[HttpServletResponseWrapper].getName + "_headerWarning")
```

Comment:
I used the underscore as the dot typically indicates level. This is at the same level, but is a special logger of this same class.

I updated the String concatenation to String interpolation. It only adds one char in this case and it is more inline with the style of this class.